Formula/yle-dl.rb

```diff
@@ -1,8 +1,8 @@
 class YleDl < Formula
   desc "Download Yle videos from the command-line"
   homepage "https://aajanki.github.io/yle-dl/index-en.html"
-  url "https://github.com/aajanki/yle-dl/archive/20210704.tar.gz"
-  sha256 "775b6a6650d684f63cee1326e1952fcfa02a683ea656fde00d3ffa8038e8af06"
+  url "https://github.com/aajanki/yle-dl/archive/20210808.tar.gz"
```

Comment:
Have we ever tried using PyPI URL?
https://files.pythonhosted.org/packages/fa/bb/3e89fa9214d97d2d590fbde45c1ad9e9975aa5dc69a86f7d2d8d3fa20c85/yle-dl-20210808.tar.gz

Reply:
yeah, I dont think we tried it before, IIRC this might be the upstream did not publish artifact to pypi before?